.jenkins

```diff
@@ -1,48 +1,30 @@
 #!/usr/bin/env groovy
 
 pipeline {
-
-  agent any
-
+  agent {
+    label 'golang-next'
```

Comment:
Fear not, the golang-next agent will stick around until we've verified the other golang projects build on it, then it'll become the golang builder (but still tagged with golang-next until we need a new golang-next builder)